mypyc/lib-rt/str_ops.c

```diff
@@ -102,8 +102,13 @@ PyObject *CPyStr_GetSlice(PyObject *obj, CPyTagged start, CPyTagged end) {
     }
     return CPyObject_GetSlice(obj, start, end);
 }
+
 /* Check if the given string is true (i.e. it's length isn't zero) */
 bool CPyStr_IsTrue(PyObject *obj) {
     Py_ssize_t length = PyUnicode_GET_LENGTH(obj);
     return length != 0;
 }
+
+Py_ssize_t CPyStr_Size_size_t(PyObject *str) {
+     return PyUnicode_GET_LENGTH(str);
```

Comment:
I think that you need to call `PyUnicode_READY` first and check for a negative return before calling `PyUnicode_GET_LENGTH`. This means that the function can fail.